Sources/Brave/Frontend/Shields/ShieldsViewController.swift

```diff
@@ -326,7 +327,21 @@ class ShieldsViewController: UIViewController, PopoverContentComponent {
   @objc private func tappedSubmitReportingButton() {
     if let url = url {
       Task { @MainActor in
-        await WebcompatReporter.reportIssue(on: url)
+        let domain = Domain.getOrCreate(forUrl: url, persistent: true)
```

Comment:
Does persistent persist the tab or get a persistent tab. I forgot. Because I guess I should just use false?

Reply:
I didnt understand what you mean there 😄 but just check if the active tab is private or not and dont send anything if it is private.

Reply:
We send the report in private mode because its all opt in (user initiates this report). However I don't know if we're supposed to pass false for `persistent` on private mode.